src/client/sagas/fetch/index.js

```diff
@@ -22,32 +22,40 @@ export function* fetchSaga(
   return null
 }
 
-export function* normalizeSaga(data: Object): Generator<any, any, any> {
-  if (!data) {
+export const transformMutationResponse = (response: Object) => {
+  let entities = response.data || response
```

Comment:
In a transformation would we ever have a data element in the response, would it always be response.operationName?